clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -85,6 +85,10 @@ class Util {
   /// property_list class.
   static bool isPropertyListType(const QualType &Ty);
```

Comment:
According to the new version of the spec (https://github.com/intel/llvm/pull/2439) `property_list` will no longer be used to specialize an accessor, hence no need to handle both `PropertyList` and `AccessorPropertyList`. This comment is applied to the code below as well.